maintain.go

```diff
@@ -66,6 +67,11 @@ func (certCache *Cache) maintainAssets(panicCount int) {
 	ctx, cancel := context.WithCancel(context.Background())
 	defer cancel()
 
+	// since checking certs for renewal is done when they're loaded, we don't need
+	// to do that until the first tick; but OCSP staples aren't necessarily refreshed
+	// at load-time, so we do this real quick before the first tick (see #4191)
+	certCache.updateOCSPStaples(ctx)
```

Comment:
How will this behave with the recursion with panics? Will it make this happen too often in a tight loop if something goes wrong?

Reply:
Good question.

I guess; but I've yet to see that happen more than once ever, and it's a pretty lightweight function (I am not even sure there's any I/O in the nominal case). The panic recursion only happens at most 10 times.